### PR TITLE
Add support for realsense 3D camera

### PR DESCRIPTION
**Pull request by lamlaaaam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,14 +12,28 @@ catkin_make
...
 
 ```xml
 <arg name="world_name" value="$(find turtlebot3_ramp_sim)/worlds/your_world_here.world"/>
 ```
+
 Launch the simulation
+
 ```ros
-roslaunch turtlebot3_ramp_sim ttb3_velodyne.launch
+roslaunch turtlebot3_ramp_sim ttb3_(sensor name).launch
```
Comment (aarushg22) [inline README.md]: you can make this something like:
```
export PERCEP_SENSOR="hokuyo"
roslaunch turtlebot3_ramp_sim ttb3_$PERCEP_SENSOR.launch
```
or you can also make a common launch file and add arguments inside the launch file to trigger various other launch files

File: README.md [hunk 2]
```diff
@@ -12,14 +12,28 @@ catkin_make
...
 ```
+
 You should see your world in Gazebo and the visualization of the LIDAR in RVIZ.
 
+## Specific instructions for RealSense attachment
+In order to run the simulation with RealSense camera, the Gazebo plugin for D435 is required.
+
+```
+cd ~/catkin_ws/src
+git clone https://github.com/pal-robotics/realsense_gazebo_plugin.git
```
Comment (aarushg22) [inline README.md:34]: Is there a binary package for this which can be directly installed using an apt install command?

**Review by aarushg22 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (lamlaaaam) [reply README.md:34]: Don't think there's a binary package for the Gazebo plugin, only for the physical sensors.

File: README.md (hunk 1 above)
Comment (lamlaaaam) [reply README.md]: Combined into a single launch file, updated README.

**Review by aarushg22 — APPROVED** [review]

**Pull request merged**